src/Auth/BaseAuthenticate.php

```diff
@@ -1,4 +1,5 @@
 <?php
+declare(strict_types = 1);
```

Comment:
I would move this up into the same line, and instead add a newline underneath :)